otelcollector/deploy/chart/prometheus-collector/templates/prometheus-collector-deployment.yaml

```diff
@@ -213,6 +213,9 @@ spec:
                 - NET_ADMIN
                 - NET_RAW
         {{- end }}
+      {{- if .Values.deploymentPriorityClass }}
```

Comment:
Ori, can you add this to our daemonset as well ?

Reply:
Added